src/app/account.service.ts

```diff
@@ -80,7 +81,10 @@ export class AccountService {
       );
       let encryptedMessagingKeyRandomness: string | undefined;
       if (privateUser.messagingKeyRandomness) {
-        encryptedMessagingKeyRandomness = this.cryptoService.encryptSeedHex(privateUser.messagingKeyRandomness, hostname);
+        encryptedMessagingKeyRandomness = this.cryptoService.encryptSeedHex(
```

Comment:
just fyi, a lot of this (like this change) is just formatting noise from formatting the file, which is annoying, but it's not a real change.